src/benchmarks/gc/src/commonlib/get_built.py

```diff
@@ -51,13 +51,13 @@ def _get_os_name() -> str:
 
 
 def is_arm() -> bool:
-    return processor().startswith("ARM")
+    return machine().find("ARM") != -1
 
 
 def _get_platform_name() -> str:
     if is_arm():
-        # TODO: detect arm32 vs 64
-        return "arm64"
+        # On ARM, the machine() function returns the exact name we need here.
+        return machine().lower()
     else:
         p = processor()
```

Comment:
How about x86?

Reply:
To add another architecture here, we would need to see how Python shows the output of `processor()` and `machine()` on said machine. I'm not sure whether Andy tested this and got `x86_64` as a result. This change would probably be better off in another PR.

Reply:
I think the older version of python would return x86_64. 
agree that checking for x86 should be done in a separate PR.